bridge-school/bridge-applications-8-frontend
# Task 1 Add Cohort Name

**Ticket** :
- https://trello.com/c/3nMZKJoz/11-admin-create-application-grouppng

**Notes**: 
- Moved components to custom form components - I think this is the way to go since I think this will be needed for the student application views. Goal here is that we can reuse all these elements again there.  I could be wrong on this/ approach. Open to suggestions here. 
- Added the required translations.
- I've styled the select option just using CSS - this came from https://css-tricks.com/styling-a-select-like-its-2019/ Also, I did this here since I was styling the input field and label anyways.  Also, I like styling things 🤷‍♀️

**QA**: 
- Check that the input element exists
- Check that the option element exists
- Check elements have an error state
- Check that elements are using the correct English text 

**Screen shot**: 
<img width="1280" alt="Screen Shot 2019-09-08 at 6 46 07 PM" src="https://user-images.githubusercontent.com/14845428/64495508-24a98800-d269-11e9-8b1a-1008a77a0ae0.png">
